radio/src/targets/taranis/telemetry_driver.cpp

```diff
@@ -91,17 +91,20 @@ static uint8_t rxByte;
 // single bit length expresses in half us
 static uint16_t bitLength;
 static uint16_t probeTimeFromStartBit;
-
+uint32_t oldPriority = 0xFFFF;
 void telemetryPortInvertedInit(uint32_t baudrate)
 {
-  if (baudrate == 0) {
+  //TODO:
+  // - handle conflict with HEARTBEAT disabled for trainer input...
+  // - probably need to stop trainer input/output and restore after this is closed
+  // - There is no need to stop trainer - we need to just enable EXT IRQ if not enabled by HEARTBEAT handling
+  //   see code below, but it is always nescesary to configure EXTI_Line for software uart.
 
-    //TODO:
-    // - handle conflict with HEARTBEAT disabled for trainer input...
-    // - probably need to stop trainer input/output and restore after this is closed
-#if !defined(TELEMETRY_EXTI_REUSE_INTERRUPT_ROTARY_ENCODER) && !defined(TELEMETRY_EXTI_REUSE_INTERRUPT_INTMODULE_HEARTBEAT)
-    NVIC_DisableIRQ(TELEMETRY_EXTI_IRQn);
-#endif
+  if (baudrate == 0) {
+    if (oldPriority != 0xFFFF) {
```

Comment:
needed?

Reply:
@bsongis Comments improved, typos fixed - up to you if you want to keep it.

Reply:
I meant: is it needed to keep the oldPriority in memory to set it again when we don't need the high priority anymore? Because we should know which is the value of "oldPriority", it should be a constant, do I miss anything?